cmd/juju/commands/migrate.go

```diff
@@ -90,6 +113,41 @@ func (c *migrateCommand) Init(args []string) error {
 	return nil
 }
 
+// Run implements cmd.Command.
+func (c *migrateCommand) Run(ctx *cmd.Context) error {
+	spec, err := c.getMigrationSpec()
+	if err != nil {
+		return err
+	}
+	modelName, err := c.ModelName()
+	if err != nil {
+		return errors.Trace(err)
+	}
+	uuids, err := c.ModelUUIDs([]string{modelName})
+	if err != nil {
+		return errors.Trace(err)
+	}
+	spec.ModelUUID = uuids[0]
+	if err := c.checkMigrationFeasibility(spec); err != nil {
+		return err
```

Comment:
Trace all these too?

Reply:
Not sure if trace here would improve visibility in case of an error (the interesting bits that can fail within the check code do use trace though). Also, the following lines don't use tracing either. ¯\_(ツ)_/¯

Reply:
There is a feature flag that prints out the error stack on error, which _could_ be useful. Not essential but good practice I think.

Reply:
Pushed https://github.com/juju/juju/pull/10092/commits/7811ea242e51587621727af1664e6b0fe9f439a3 to add tracing.